unit1/lesson13.rst

```diff
@@ -0,0 +1,36 @@
+Unit 1 - Lesson 2 Learning Objectives
+=====================================
+
+Goals
+-----
+
+- Add styles to static HTML pages using CSS.
+- Work with Git to version control personal work.
+- Learn how Git can be used to manage teamwork.
+
+Objectives
+----------
+
+CSS
+^^^
+
+- Explain how CSS allows for separation of display and document structure
+- Describe the 3 locations that style rules may be placed: external, document, inline
```

Comment:
As an addendum to this objective, or a new objective itself, it seems like students should understand when one location may be preferred over another for style rules.

Reply:
I like that as an addendum. I think it will flow nicely!